new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -573,20 +596,291 @@ public void testDomainInPvUsingWdt() {
         .isTrue();
   }
 
+
```

Comment:
For a dynamic config.xml change (e.g. changing  cluster size) updating the introSpectVersion will initiate a simultaneous re-start.

For a non-dynamic config.xml change (e.g. changing  admin server port ) updating the introSpectVersion will initiate a rolling re-start.

Is this assumption is true ?

Reply:
I don't think so. As far as I know, the re-introspect behavior is the same with either type of change, and an introspectVersion change is not expected to trigger a roll.  @russgold ^^^

Reply:
> For a dynamic config.xml change (e.g. changing cluster size) updating the introSpectVersion will initiate

It will only start the new pod if replica is also changed, won't restart existing pods